manifests/plugin/dynflow.pp

```diff
@@ -71,4 +74,13 @@
       enable => true,
     }
   }
+
+  if $::foreman_proxy::plugin::dynflow::open_file_limit and $::osfamily == 'RedHat' {
```

Comment:
I'd actually put this at the same indent level as ```service``` with the additional open_file_limit if. There's also no need to use this from the global scope since it's all in the local scope as well. That means you can just use ```$open_file_limit```.